lib/OptimizationBBO/src/OptimizationBBO.jl

```diff
@@ -85,12 +111,19 @@ function SciMLBase.__solve(prob::SciMLBase.OptimizationProblem, opt::BBO,
 
     opt_args = __map_optimizer_args(prob, opt, callback = _cb, maxiters = maxiters,
                                     maxtime = maxtime, abstol = abstol, reltol = reltol;
-                                    kwargs...)
+                                    verbose = verbose, kwargs...)
 
     opt_setup = BlackBoxOptim.bbsetup(_loss; opt_args...)
 
     t0 = time()
+    
+    if progress
+        # Set progressbar to 1 to finish it
+        Base.@logmsg(Base.LogLevel(-1), msg, progress=1, _id=:OptimizationBBO)
```

Comment:
`msg` is not defined here. Should it be `""`?
Also, shouldn't this be after the `bboptimize` call?

Reply:
I've added a commit to do that